.markdownlinkcheck.json

```diff
@@ -1,11 +1,14 @@
 {
     "ignorePatterns": [],
-    "httpHeaders": [{
-        "urls": [
-            "https://", "http://"
-        ],
-        "headers": {
-            "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:64.0) Gecko/20100101 Firefox/64.0"
+    "httpHeaders": [
```

Comment:
this file is deprecated and should be removed soon. It was used by gendocs.sh (which was used for travis)

Reply:
So the files using the old config file are:
- `Apply_Link_Check.sh`
- `gendocs.sh`
- `gendocsLocal.sh`

And these are mentioned in the `./tools/README.md` file.
I can remove them all.

What instead could be done is a reference in the `CONTRIBUTING.md` on how to check for linting issues and that links are working.
What do you think?